cost-model/src/block_cost_limits.rs

```diff
@@ -53,3 +54,16 @@ pub const fn simd_0207_block_limits() -> (u64, u64, u64) {
         MAX_VOTE_UNITS,
     )
 }
+
+/// Return the block limits that will be used upon activation of SIMD-0256.
+/// Returns as
+/// (account_limit, block_limit, vote_limit)
+// ^ Above order is used to be consistent with the order of
+//   `CostTracker::set_limits`.
+pub const fn simd_0256_block_limits() -> (u64, u64, u64) {
```

Comment:
@t-nelson - this probably makes your eyes bleed... mea culpa.

Given we're going to keep doing this several more times, what would you like this to look like?

Reply:
Given we plan to continue this chain, maybe it's better to not use a tuple, and just define a new structure:
```rust
pub struct BlockLimits {
    account_cost_limit: u64,
    block_cost_limit: u64,
    vote_cost_limit: u64,
}
```

refactor `CostTracker::set_limit` to take this object and store it internally in the `CostTracker` as well.

Reply:
if we plan to backport this, would prefer to keep the simple tuple. then do backport only on master

Reply:
let's still get Trent's thoughts though - may be different than mine

Reply:
i'd really like to stop kicking the can on this shit. get the value out of the code and on chain (requires its own simd)

don't care for this one, but i'm going to neg every future cu bump that references the constant